MAUI/Picker/populating-items.md

```diff
@@ -0,0 +1,218 @@
+    cityName.Add("Chennai");
+    cityName.Add("Mumbai");
+    cityName.Add("Delhi");
+    cityName.Add("Kolkata");
+    cityName.Add("Bangalore");
+    cityName.Add("Hyderabad");
+    cityName.Add("Pune");
+    cityName.Add("Ahmedabad");
+    cityName.Add("Jaipur");
+    cityName.Add("Lucknow");
+    cityName.Add("Chandigarh");
+    PickerColumn pickerColumn = new PickerColumn()
+    {
+        HeaderText = "Select City",
+        ItemsSource = cityName,
+        SelectedIndex = 1,
+    };
+    this.Picker.Add(pickerColumn);
+
+{% endhighlight %}
+{% endtabs %}
+
+## Multi-column items
+
+The picker populates the items as Multi-column based on the columns value.
+
+Collection of items can be created and assigned to a Collection based on index value, and each index is a column of picker.
+
+The following code example encapsulate how to populate Country and Cityes in each column of picker.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:sfPicker="clr-namespace:Syncfusion.Maui.Picker;assembly=Syncfusion.Maui.Picker"
+             x:Class="PickerSample.MainPage">
+
+    <sfPicker:SfPicker x:Name="Picker">
+    </sfPicker:SfPicker>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    ObservableCollection<object> countryName = new ObservableCollection<object>();
+    countryName.Add("India");
+    countryName.Add("United States");
+    countryName.Add("Canada");
+    countryName.Add("United Kingdom");
+    countryName.Add("Australia");
+    countryName.Add("Germany");
+    countryName.Add("France");
+    countryName.Add("Japan");
+    countryName.Add("China");
+    countryName.Add("Brazil");
+    PickerColumn countryColumn = new PickerColumn()
+    {
+        HeaderText = "Select Country",
+        ItemsSource = countryName,
+        SelectedIndex = 1,
+    };
+    ObservableCollection<object> cityName = new ObservableCollection<object>();
+    cityName.Add("Chennai");
+    cityName.Add("Mumbai");
+    cityName.Add("Delhi");
+    cityName.Add("Kolkata");
+    cityName.Add("Bangalore");
+    cityName.Add("Hyderabad");
+    cityName.Add("Pune");
+    cityName.Add("Ahmedabad");
+    cityName.Add("Jaipur");
+    cityName.Add("Lucknow");
+    cityName.Add("Chandigarh");
+    PickerColumn cityColumn = new PickerColumn()
+    {
+        HeaderText = "Select City",
+        ItemsSource = cityName,
+        SelectedIndex = 1,
+    };
+    this.Picker.Columns.Add(countryColumn);
+    this.Picker.Columns.Add(cityColumn);
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set items text style customization
+
+The .NET MAUI Picker control, both selected and unselected items text color, font size, font family, font attributes can be customize as shown in the following code.
```

Comment:
The .NET MAUI Picker control, selected and unselected items, text color, font size, font family, and font attributes can be customized as shown in the following code.

Reply:
Content modified